src/config.ts

```diff
@@ -18,3 +20,38 @@ export const nodeImportMapPath = join(root, "node.importmap");
 export const cache = join(root, ".cache");
 const map = existsSync(nodeImportMapPath) ? JSON.parse(readFileSync(nodeImportMapPath, { encoding: "utf8" })) : {};
 export const importmap = new ImportMap({ rootUrl: import.meta.url, map });
+
+interface OptionDefinition {
+  type?: string;
+  alias?: string;
+  default?: any;
+}
+
+interface Options {
+  args: string[];
+  options: Record<string, OptionDefinition>;
+}
+
+function parseArgs({ args, options: optionDefinitions }: Options) {
+  const parsedArgs: Record<string, string | boolean> = {};
+
+  for (let i = 0; i < args.length; i++) {
```

Comment:
This could be a map and then reduce, right? 
What is the intent for using the for loop here?